benchmark/micro/pushdown/pwmj_inequality_pushdown.benchmark

```diff
@@ -0,0 +1,23 @@
+# name: benchmark/micro/pushdown/pwmj_inequality_pushdown.benchmark
+# description: Show the performance of using pushdown with PieceWiseMergeJoin
+# group: [pushdown]
+
+name PieceWiseMergeJoinInequalityPushdown
+group micro
+subgroup pushdown
+
+require tpch
+
+argument sf 10
+
+cache tpch_sf${sf}.duckdb
+
+load
+CALL dbgen(sf=${sf});
```

Comment:
Not a blocker for this PR, but just a note - all of this can be replaced with the new `include` syntax:

```sql
include benchmark/tpch/tpch_load.benchmark.in
```